src/spec.ts

```diff
@@ -77,7 +77,7 @@ export interface GenericUnitSpec<M, E extends Encoding> extends BaseSpec {
   encoding: E;
 }
 
-export type UnitSpec = GenericUnitSpec<Mark, Encoding>;
+export type UnitSpec = GenericUnitSpec<Mark | MarkDef, Encoding>;
```

Comment:
@domoritz  Quick question -- should we rename Mark to MarkType.

I'm leaning toward no because we use it in too many places and I don't wanna make it too long.